src/test/kotlin/nl/info/client/pabc/PabcClientServiceTest.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.client.pabc
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.mockk.checkUnnecessaryStub
+import io.mockk.clearMocks
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.slot
+import nl.info.client.pabc.model.generated.GetApplicationRolesRequest
+import nl.info.client.pabc.model.generated.GetApplicationRolesResponse
+
+class PabcClientServiceTest : BehaviorSpec({
+
+    val pabcClient = mockk<PabcClient>()
+
+    val roles = listOf(
+        "behandelaar",
+        "coordinator",
+        "beheerder",
+        "recordmanager",
+        "raadpleger"
+    )
+
+    val mockResponse = GetApplicationRolesResponse()
```

Comment:
We normally create fixtures for creating such mock objects in separate fixture files. E.g. `createGetApplicationRolesResponse()` in this case just to have that in one place. For now it doesn't matter too much yet since we don't really do much with the response yet.